web/settings/settings.go

```diff
@@ -84,6 +88,37 @@ func (h *HTTPHandler) warnings(c echo.Context) error {
 	return jsonapi.DataErrorList(c, warnings...)
 }
 
+// postEmail handle POST /settings/email
```

Comment:
Can you document this endpoint in https://github.com/cozy/cozy-stack/blob/master/docs/settings.md?